### PR TITLE
Propose Emotable Extension for Non-Fungible Tokens

### PR DESCRIPTION
**Pull request by ThunderDeliverer**
This EIP draft proposal includes RMRK team's work on developing an NFT standard where an NFT can be reacted to using Unicode emojis.

### CODE REVIEW

File: EIPS/eip-6381.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+eip: 6381
+title: Emotable Extension for Non-Fungible Tokens
+description: An interface for Non-Fungible Tokens extension allowing for reacting to them using Unicode emojis.
```
Comment (SamWilsn) [inline EIPS/eip-6381.md:4]: This seems a bit more succinct:

```suggestion
description: React to Non-Fungible Tokens using Unicode emojis.
```

File: EIPS/eip-6381.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+Current NFT market heavily relies on previous values the token has been sold for, the lowest price of the listed token and the scarcity data provided by the marketplace. There is no real time indication of admiration or desirability of a specific token. Having the ability for users to emote to the tokens adds the possibility of potential buyers and sellers gageing the value of the token based on the impressions the token has collected.
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+```solidity
+/// @title EIP-6381 Emotable Extension for Non-Fungible Tokens
+/// @dev See https://eips.ethereum.org/EIPS/eip-5773
```
Comment (SamWilsn) [inline EIPS/eip-6381.md:50]: Wrong EIP number?

File: EIPS/eip-6381.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+/// @dev Note: the ERC-165 identifier for this interface is 0xf8d6854d.
+
+pragma solidity ^0.8.16;
+
+interface IRMRKEmotable is IERC165 {
```
Comment (SamWilsn) [inline EIPS/eip-6381.md:55]: Dosen't match the interface in the reference.

File: EIPS/eip-6381.md (hunk 2 above)
Comment (steven2308) [reply EIPS/eip-6381.md]: Fixed

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**